### PR TITLE
feat(ins): add instructions for backup via cli

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: compute/instances/api-cli/creating-backups.mdx [hunk 1]
```diff
@@ -22,48 +22,97 @@ The Backup feature is used to back up your Instance data. It creates an image of
...
-A backup is created using the command:
-```
-POST https://api.scaleway.com/instance/v1/zones/<region>/servers/<uid>/action
-{"action":"backup"}
-```
+<Tabs id="createbackup">
 
-Replace `<region>` with the geographical region of the Instance. This value can be `fr-par-1` for Instances located in Paris, for example. Replace `<uid>` with the unique identifier of the Instance.
+  <TabsTab label="CLI">
+    ## Creating a backup using the CLI
```
Comment (SamyOubouaziz) [inline compute/instances/api-cli/creating-backups.mdx:28]: Did not think about it, but putting H2 anchors inside tabs breaks the "jump to" menu (anchors that are not visible, i.e. in the active tab cannot be accessed from the "jump to" menu.

An approach could be:

```
## Creating a backup
<Tabs>
<CLI Tab />
<API Tab />
</Tabs>
## Deleting a backup
<Tabs>
<CLI Tab />
<API Tab />
</Tabs>
```

File: compute/instances/api-cli/creating-backups.mdx [hunk 2]
```diff
@@ -13,57 +13,89 @@ categories:
...
+    An image contains one snapshot for each volume of the Instance. These snapshots are visible within the image response as `root_volume` and `extra_volumes` fields.
+  </TabsTab>
+
+  <TabsTab label="API">
+    A backup is created using the [Instances API](/developers/api/instance/) with the following command:
+    ```
+    POST https://api.scaleway.com/instance/v1/zones/<region>/servers/<uid>/action 
+    {"action":"backup"}
+    ```
+    Replace `<region>` with the geographical region of the Instance. This value can be `fr-par-1` for Instances located in Paris, for example. Replace `<uid>` with the unique identifier of the Instance.
```
Comment (RoRoJ) [inline compute/instances/api-cli/creating-backups.mdx:56]:
```suggestion
    Replace `<region>` with the geographical region of the Instance. For Instances located in Paris, for example, this value would be `fr-par-1`. Replace `<uid>` with the unique identifier of the Instance.
```

File: compute/instances/api-cli/creating-backups.mdx [hunk 3]
```diff
@@ -13,57 +13,89 @@ categories:
...
+    By default, the name of the image is built according to the name of the server and the date:
+    ```
+    image_<servername>_YYYY-MM-DD_hh-mm.
+    ```
+    Alternatively, you can specify a name for the image in the request:
+    ```
+    POST https://api.scaleway.com/instance/v1/zones/<region>/servers/<id>/action
+    { "action":"backup", "name":"myimagename" }
+    ```
+    A backup request will create an Image object (Visible in the `href_result`: `/images/<image_uuid>`)
```
Comment (RoRoJ) [inline compute/instances/api-cli/creating-backups.mdx:66]:
```suggestion
    A backup request will create an Image object (visible in the `href_result`: `/images/<image_uuid>`)
```

**Review by RoRoJ — APPROVED** [review]

File: compute/instances/api-cli/creating-backups.mdx [hunk 4]
```diff
@@ -13,57 +13,89 @@ categories:
...
+    By default, the name of the image is built according to the name of the server and the date:
+    ```
+    image_<servername>_YYYY-MM-DD_hh-mm.
+    ```
+    Alternatively, you can specify a name for the image in the request:
+    ```
+    POST https://api.scaleway.com/instance/v1/zones/<region>/servers/<id>/action
+    { "action":"backup", "name":"myimagename" }
+    ```
+    A backup request will create an Image object (visible in the `href_result`: `/images/<image_uuid>`)
```
Comment (ldecarvalho-doc) [inline compute/instances/api-cli/creating-backups.mdx:66]:
```suggestion
    A backup request will create an image object (visible in the `href_result`: `/images/<image_uuid>`)
```

File: compute/instances/api-cli/creating-backups.mdx [hunk 5]
```diff
@@ -13,57 +13,89 @@ categories:
...
+      * Replace `server-id` with the unique identifier of the Instance.
+      * Replace `fr-par-1` with the Availability Zone your Instance is deployed in.
+    </Message>
+    <Message type="tip">
+      By default, the name of the image is built according to the name of the server and the date. You can specify a name for the image in the request:
+      ```sh
+      scw instance server backp server-id zone=fr-par-1 name=myimagename
+      ```
+    </Message>
+    A backup request will create an Image object. You can view it using:
```
Comment (ldecarvalho-doc) [inline compute/instances/api-cli/creating-backups.mdx:43]:
```suggestion
    A backup request will create an image object. You can view it using:
```

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**